models/discounts.go

```diff
@@ -0,0 +1,43 @@
+package models
+
+import (
+	"time"
+
+	"github.com/mundipagg/boleto-api/util"
+)
+
+type Discount struct {
+	Date       string  `json:"date,omitempty"`
+	Percentage float64 `json:"percentage,omitempty"`
+}
+
+func (d *Discount) HasValidDate() error {
```

Comment:
Opa, @kayquecoelho , geralmente os prefix has e is voltam bool, dai talvez vale alterarmos o nome desse método 😄 
```suggestion
func (d *Discount) ValidateDate() error {
```

Reply:
boa! Vou alterar aqui